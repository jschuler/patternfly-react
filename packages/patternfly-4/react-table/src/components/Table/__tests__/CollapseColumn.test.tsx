// Auto-generated do not edit

import * as React from 'react';
import { shallow } from 'enzyme';
import { CollapseColumn } from '../CollapseColumn';

describe('CollapseColumn test', () => {
  it('CollapseColumn should match snapshot', () => {
    const view = shallow(
      <CollapseColumn
        id={undefined}
				className={"''"}
				children={<>ReactNode</>}
				onToggle={undefined}
				isOpen={undefined}
      />);
    expect(view).toMatchSnapshot();
  });
});
