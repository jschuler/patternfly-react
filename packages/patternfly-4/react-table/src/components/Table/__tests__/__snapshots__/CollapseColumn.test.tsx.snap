// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CollapseColumn test CollapseColumn should match snapshot 1`] = `
<Fragment>
  ReactNode
</Fragment>
`;
